### ISSUE TITLE
Join with Fdroid?

### ISSUE THREAD

**Issue by haarp** [droidbreak/droidbreak.github.io#48]
Hey,

this is a really nice list you have here. Clear layout, great information. Have you ever considered joining forces with Fdroid? I think that would make much more sense than running a list in parallel. On the Fdroid side, their site and representation could use a refresher, and on your side, you have lots of useful information and a great layout. A merger would be win-win.

Cheers!

**Comment by repat** [comment]
Hi @haarp,

FDroid is by nature open by nature and everybody can submit projects. Droidbreak is a moderated list that only takes the best apps in their respective categories to replace proprietary apps.

**Comment by haarp** [comment]
That doesn't have to be mutually exclusive. One could easily have a prominent "best of" section in Fdroid :)

**Comment by repat** [comment]
That's true. Unfortunately not all projects on droidbreak are available on FDroid, e.g. [Signal](https://play.google.com/store/apps/details?id=org.thoughtcrime.securesms)

**Comment by haarp** [comment]
That is true, unfortunately. How about a "coming soon" tag?

There's still major overlap between your list and apps on Fdroid. It just feels like it would be a shame if there is a lot of duplicated work, or if information that users could be interested in is split over several places.
